website/static/js/stripe-internal-csat.js

```diff
@@ -0,0 +1,19 @@
+(() => {
+  if (document.querySelector('#csat-extension-config') != null) {
+    return;
+  }
+
+  const selector = '.mainContainer.docsContainer > .wrapper';
+  const main = document.querySelector(selector);
+  if (main != null) {
+    const csatDiv = document.createElement('div');
+    csatDiv.id = 'csat-extension-config';
+    const config = {
+      embedded: true,
+      prompt: 'Feedback for #sorbet?',
+      notify: '#sorbet',
+    };
+    csatDiv.setAttribute('data-config', JSON.stringify(config));
+    main.appendChild(csatDiv);
+  }
+})();
```

Comment:
Not-very-frontend-savvy question: why is this div invisible? Is it just because it has no children?

Reply:
all divs are invisible unless they have a height, width, and content